tests/e2e/Trustline.test.ts

```diff
@@ -385,5 +392,108 @@ describe('e2e', () => {
         expect(tl1.trustline.getAll(networkWithoutInterestRates.address)).to.eventually.be.an('array')
       })
     })
+
+    describe('#prepareClose()', () => {
+      const given = 1000
+      const received = 1000
+      const transferAmount = 100
+
+      before(async () => {
+        // Construction of three accounts as clique.
+        const triangle = [[tl1, tl2], [tl2, tl3], [tl3, tl1]]
+
+        // Establish all trustlines in the triangle.
+        for (let trustline of triangle) {
+          // Get the both users for this trustline.
+          const [a, b] = trustline
+
+          // Prepare trustline update transaction from a to b.
+          const { rawTx: rawUpdateTx } = await a.trustline.prepareUpdate(
+            networkWithoutInterestRates.address,
+            b.user.address,
+            given,
+            received
+          )
+
+          // Prepare trustline accept transaction from b to a.
+          const { rawTx: rawAcceptTx } = await b.trustline.prepareAccept(
+            networkWithoutInterestRates.address,
+            a.user.address,
+            given,
+            received
+          )
+
+          // Sign and relay prepared transactions.
+          await Promise.all([
+            a.trustline.confirm(rawUpdateTx),
+            b.trustline.confirm(rawAcceptTx)
+          ])
+
+          // Wait for txs to be mined.
+          await wait()
+        }
+
+        // Manipulate the balance between the user to settle the trustline later on.
+        const { rawTx: rawPaymentTx } = await tl1.payment.prepare(
+          networkWithoutInterestRates.address,
+          tl2.user.address,
+          transferAmount
+        )
+        await tl1.payment.confirm(rawPaymentTx)
+        await wait()
+      })
+
+      it('should be possible to prepare a close in the correct direction.', async () => {
+        // Send the prepare settle to the relay, expecting a valid path exists.
+        const closeTx = await tl1.trustline.prepareClose(networkWithoutInterestRates.address, tl2.user.address)
+        expect(closeTx).to.have.keys(['rawTx', 'value', 'ethFees', 'maxFees', 'path'])
+        expect(closeTx.path).to.include(tl3.user.address)
+        expect(closeTx.ethFees).to.have.keys(['raw', 'value', 'decimals'])
+        expect(closeTx.maxFees).to.have.keys(['raw', 'value', 'decimals'])
+        expect(closeTx.value).to.have.keys(['raw', 'value', 'decimals'])
+        expect(closeTx.value.value).to.equal(transferAmount.toString())
+      })
+
+      it('should not be possible to prepare a close in the wrong direction.', async () => {
+        // Send the prepare settle to the relay, expecting that a 501 HTTP status code get responded.
+        try {
+          await tl2.trustline.prepareClose(networkWithoutInterestRates.address, tl1.user.address)
+        } catch (err) {
+          expect(err).to.be.an('error')
```

Comment:
I dont think we should have this test here. This is currently not implemented, but should be in the future. This reads as if we would expect it from beeing not working. 
I would say we either remove the test or make it a proper test but skip it for now.